### PR TITLE
Feature: Certified Developer Program file translated to PT-BR

### PR DESCRIPTION
**Pull request by pedroSoaresll**
@glaucia86 
@cezaraugusto 
@Drieger 
@khaosdoctor 
@maugravena 

Oi galera, fiz pra gente a tradução do arquivo `locale / pt-br / blog / announcements / nodejs-certified-developer-program.md` para portugues.

OBS: peço ajuda para vocês para me ajudares a tornar a descrição dessa PR relevante e bem documentada seguindo os padrões aqui da comunidade, e caso eu tenha esquecido de algo no PR estarei a disposição para fixar. Obrigado.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # [nodejs.org](https://nodejs.org/)
 
+
```
Comment (Trott) [inline README.md]: Unrelated whitespace change in an unrelated file?

Comment (pedroSoaresll) [reply README.md]: really, i'll remove it. I don't know what it happened rs

Comment (pedroSoaresll) [reply README.md]: done, thank you

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
+slug: nodejs-certified-developer-program.md
+layout: blog-post.hbs
+---
+
+_Fundação Node.js a ser lançada
+fornecedor-neutro programa de certificação para a plataforma que mais cresce_
+
+**SAN FRANCISCO, Jan. 26, 2017** — [A Fundação Node.js](https://foundation.nodejs.org/), um liderado pela comunidade e
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:12]: There's something wrong here, "um liderado pela ..." this translation asks for more information

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: Os Parceiros da Fundação Node.js com a Fundação Linux no Novo Programa de Certificação Node.js
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
+slug: nodejs-certified-developer-program.md
+layout: blog-post.hbs
+---
+
+_Fundação Node.js a ser lançada
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:9]: This phrase makes no sense...

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+fornecedor-neutro programa de certificação para a plataforma que mais cresce_
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:10]: This is another phrase that doesn't make sense. It needs to be reviewed as one.

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+**SAN FRANCISCO, Jan. 26, 2017** — [A Fundação Node.js](https://foundation.nodejs.org/), um liderado pela comunidade e
+apoiado pela indústria consórcio para avançar no desenvolvimento da plataforma Node.js,
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:13]: Another phrase with complicated meaning. "indústria consórcio" is something that does not make sense here

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+hoje anunciamos o desenvolvimento do programa inaugural de certificação Node.js
+destinado a estabelecer uma competência de linha de base no Node.js.
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md]: Common translators tend to translate `baseline` to `Linha de base` which is wrong. 

```suggestion
destinado a estabelecer um parâmetro base para as competências no Node.js.
```

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+Node.js é uma das linguagens de programação mais populares com mais de 4.5 milhões
+de usuários ativos por mês. Enquanto Node.js é cada vez mais difundido hoje com
+empresas de todos os tamanhos, organizações que desejam expandir o uso de Node.js
+muitas vezes lutam ao treinar desenvolvedores Java e recrutar
+novos talentos.
+
+“A fundação Node.js, com ajuda ded incríveis membros da comunidade e principais
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md]:
```suggestion
“A fundação Node.js, com ajuda de incríveis membros da comunidade e principais
```

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+especialistas, está criando um programa abrangente de certificação que amplia o
+funil de conhecimento qualificado do Node.js disponível. Seja trabalhando em ambientes
+corporativos ou como consultores individuais, aqueles que se tornam desenvolvedores
+certificadores Node.js serão bem posicionados a começar em todo vapor como um desenvolvedor
+Node.js, possuindo habilidades que estão em alta demanda,” disse Tracy Hinds,
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md]:
```suggestion
Node.js, possuindo habilidades que estão em alta demanda.” disse Tracy Hinds,
```

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+gerente de comunidade educacional para a Fundação Node.js.
+
+O programa de desenvolvedor certificado Node.js, que está sendo desenvolvido com a contribuição dos principais
+especialistas Node.js e contribuidores, deverá estar disponível no segundo trimestre de 2017. O programa proverá um quadro geral
+de competência em Node.js,
+ajudando empresas a identificar rapidamente engenheiros Node.js qualificados, enquanto
+fornece desenvolvedores, empreiteiros e consultores com um caminho para diferenciar
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:35]: I think "empreiteiros" is not quite right here

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+eles mesmos no mercado.
+
+Fundação Node.js é trabalhado em estreita colaboração com [A Fundação Linux](https://training.linuxfoundation.org/certification/why-certify-with-us) para criar o projeto
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:38]: Probably missing correct verb conjugation here

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+e processos para administrar o programa. A Fundação Linux oferece uma
+casa neutra para executar programas de treinamento e certificação, obrigado por isso
+envolvimento próximo com a comunidade de código aberto. Isso oferece vários cursos
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:41]: "obrigado por isso envolvimento próximo com a comunidade de código aberto" is also missing some conjugation, but the translation looks weird

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+online abertos (MOOCs), Incluindo uma [Introdução ao Linux](https://www.edx.org/course/introduction-linux-linuxfoundationx-lfs101x-0), [Introdução ao DevOps: Transformando e melhorando operações](https://www.edx.org/course/introduction-devops-transforming-linuxfoundationx-lfs161x); [Desenvolvendo aplicações para Linux](https://training.linuxfoundation.org/linux-courses/development-training/developing-applications-for-linux); [Kubernetes Fundamentos](https://training.linuxfoundation.org/linux-courses/system-administration-training/kubernetes-fundamentals); entre muitos outros.
+
+Candidatos ideais ao desenvolvedor certificado do Node.js são desenvolvedores iniciais de nível intermediário
+quem já pode trabalhar proficientemente em JavaScript com a plataforma Node.js.
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md]:
```suggestion
que já podem trabalhar proficientemente com JavaScript e com a plataforma Node.js.
```

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+Preços para o ritmo individual, exame on-line ainda está para ser determinado.
```
Comment (khaosdoctor) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:46]: I don't think "ritmo" is the correct translation...

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
There are a few problems due to the automatic translation process using tools like Google Translator. When using tools like this, please re-read and review the text, because these tools are prone to conjugation errors.

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md (hunk 2 above)
Comment (pedroSoaresll) [reply locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:12]: amazing @khaosdoctor, did you found some other phrase with conjugation wrong?
Or just this?

**Comment by khaosdoctor** [comment]
Hey @pedroSoaresll try to pull from the upstream branch so it can be up to date with the base branch :)

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Just conjugation and translation corrections to be applied

**Comment by pedroSoaresll** [comment]
@khaosdoctor thank you for your review, I'll review all your points o/

**Comment by alexandrtovmach** [comment]
@pedroSoaresll Thank you for contribution. We're moving translation process to Crowdin, please read these guidelines for more information:
https://github.com/nodejs/i18n/#contributing

@nodejs/nodejs-pt Please manage this PR [until **May 1, 2020**](https://github.com/orgs/nodejs/teams/translators/discussions/7)

**Review by TiagoDanin — APPROVED** [review]
👍 ~~LGTM~~

EDIT: PR wrong page, sorry :)

File: locale/pt-br/blog/announcements/nodejs-certified-developer-program.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
+---
+title: Os Parceiros da Fundação Node.js com a Fundação Linux no Novo Programa de Certificação Node.js
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
```
Comment (TiagoDanin) [inline locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:4]: Category is translated ?

Comment (khaosdoctor) [reply locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:4]: I think they are so they're shown in the repo, but I'm not sure. @Trott do you know something about it?

Comment (XhmikosR) [reply locale/pt-br/blog/announcements/nodejs-certified-developer-program.md:4]: Nope, shouldn't be translated.

**Comment by Trott** [comment]
This seems to be a long-stalled PR. I'm going to close it, but feel free to comment or re-open if it's still being worked on or is ready to be merged.
